lib/api/objectPut.js

```diff
@@ -99,19 +99,24 @@ function objectPut(authInfo, request, streamingV4Params, log, callback) {
             // have overwritten if there was an existing version or object
             const oldByteLength = objMD && objMD['content-length']
```

Comment:
Thanks for noting this. I've made some changes to address it.

Updated the callback here: https://github.com/scality/S3/pull/721/commits/94bbca2b4a5e1600d1c2f6adf33cd1d2978cd025.
Added a test in Integration: https://github.com/scality/Integration/pull/521/commits/a2f08ccd645f7138547ab410c93fc886432a365e